app/editor/package.json

```diff
@@ -2,71 +2,71 @@
   "name": "tno-editor-app",
   "description": "Today's News Online Editor application",
   "version": "0.1.0",
-  "private": true,
-  "homepage": "/app/",
+  "license": "Apache-2.0",
+  "private": false,
+  "homepage": "/",
   "dependencies": {
-    "@headlessui/react": "1.4.1",
+    "@headlessui/react": "1.4.2",
     "@react-keycloak/web": "3.4.0",
-    "@reduxjs/toolkit": "1.6.1",
-    "axios": "0.21.4",
-    "axios-mock-adapter": "1.19.0",
-    "formik": "2.2.6",
-    "history": "5.0.1",
-    "keycloak-js": "12.0.4",
+    "@reduxjs/toolkit": "1.7.1",
+    "axios": "0.24.0",
```

Comment:
Updated as many dependencies as I could.  This took far too much time to do...  So many issues.